src/main/kotlin/no/nav/tilleggsstonader/sak/arbeidsfordeling/ArbeidsfordelingService.kt

```diff
@@ -62,6 +62,12 @@ class ArbeidsfordelingService(
                 stønadstype = stønadstype,
                 oppgavetype = oppgavetype,
             )
+        if (stønadstype == Stønadstype.DAGLIG_REISE_TSR) {
```

Comment:
Hvis man plasserer denne ovenfor `lagArbeidsfordelingKritierieForPerson` så får man en "quick exit", og ikke kaller `lagArbeidsfordelingKritierieForPerson` uten å trenge svaret

Litt usikker på om det er denne metoden som brukes for opprettelse av behandle-sak oppgaver 👀

Reply:
Smart!

Denne brukes i `opprettOppgaveUtenÅLagreIRepository` som ser ut til å opprette en oppgave i gosys/oppgavesystemet. `opprettOppgaveUtenÅLagreIRepository` brukes igjen av `OpprettOppgaveTask` (Opprett oppgave i GOSYS for behandling) og `OpprettOppgaveForOpprettetBehandlingTask` (Opprett behandle sak oppgave for opprettet behandling). Trooor det skal bli rett?

Reply:
Ta gjerne en titt på hvilken metode som kaller på denne og hvordan :-)

Reply:
Hehe tror jeg skjønner nå. Flyttet logikken her: fdecb1f696583c0c4fd644ed393538e6a352743b. Tor det blir rett nå? 🤞